corehq/apps/commtrack/processing.py

```diff
@@ -66,6 +67,21 @@ def _create_model_for_stock_report(domain, stock_report_helper):
         domain=domain,
     )
 
+LedgerValues = namedtuple('LedgerValues', ['balance', 'delta'])
+
+
+def compute_ledger_values(lazy_original_balance, report_type, quantity):
```

Comment:
super nit picky, but it feels to me like `lazy_original_balance` should be last argument since it's the most optional